opentelemetry-otlp/examples/basic-otlp-http/src/main.rs

```diff
@@ -44,19 +44,23 @@ fn init_tracer() -> Result<sdktrace::Tracer, TraceError> {
         .install_batch(opentelemetry_sdk::runtime::Tokio)
 }
 
-fn init_metrics() -> metrics::Result<sdkmetrics::SdkMeterProvider> {
+fn init_metrics() -> Result<(), MetricsError> {
     let export_config = opentelemetry_otlp::ExportConfig {
         endpoint: "http://localhost:4318/v1/metrics".to_string(),
         ..opentelemetry_otlp::ExportConfig::default()
     };
-    opentelemetry_otlp::new_pipeline()
+    let provider = opentelemetry_otlp::new_pipeline()
         .metrics(opentelemetry_sdk::runtime::Tokio)
         .with_exporter(
             opentelemetry_otlp::new_exporter()
                 .http()
                 .with_export_config(export_config),
         )
-        .build()
+        .build();
+    match provider {
+        Ok(provider) => Ok(()),
+        Err(err) => Err(err),
+    }
```

Comment:
```suggestion
      provider.map(|_| ())
```
Should do the trick

Does OTLP pipeline already set the provider as global meter provider?

Reply:
OTLP Pipeline does that, but I consider that as a bug, and should be fixed : https://github.com/open-telemetry/opentelemetry-rust/issues/1592